packages/blade/src/components/Typography/BaseText/BaseTextTypes.ts

```diff
@@ -0,0 +1,48 @@
+import type { Theme } from '~components/BladeProvider';
+import type { DotNotationColorStringToken } from '~src/_helpers/types';
+import type { Feedback } from '~tokens/theme/theme';
+import type { AccessibilityProps } from '~utils';
+
+type FeedbackColors = `feedback.text.${DotNotationColorStringToken<
+  Theme['colors']['feedback']['text']
+>}`;
+type FeedbackActionColors = `feedback.${Feedback}.action.text.${DotNotationColorStringToken<
+  Theme['colors']['feedback'][Feedback]['action']['text']
+>}`;
+type SurfaceColors = `surface.text.${DotNotationColorStringToken<
+  Theme['colors']['surface']['text']
+>}`;
+type ActionColors = `action.text.${DotNotationColorStringToken<Theme['colors']['action']['text']>}`;
+
+export type BaseTextProps = {
```

Comment:
Can probably add JSDoc comments on non react native props so they will show up in editor